src/runtime/Object.h

```diff
@@ -722,6 +722,9 @@ class Object : public PointerValue {
     explicit Object(ExecutionState& state);
     explicit Object(ExecutionState& state, Object* proto);
 
+    enum PrototypeIsNullTag { PrototypeIsNull };
+    explicit Object(ExecutionState& state, PrototypeIsNullTag); // I added new function for reducing checking null for prototype
```

Comment:
How about adding "__" prefix and postfix for `PrototypeIsNull`?
Other enum values such as ForGlobalBuiltin, ForBuiltinConstructor use the same prefix and postfix.

Reply:
IMO adding __ at prefix, postfix means `the flag or something` is internal or users would not use that flag or something.
but I want to encourage using this function.
I think we can move __ForGlobalBuiltin__ into private area.

Reply:
Great idea!